src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -75,10 +77,10 @@ Friend Module ParserTestUtilities
 
         ' Verify Errors
         If expectedDiagnostics Is Nothing Then
-            Dim errors As New StringBuilder()
+            Dim errors = _PooledStringBuilderPool.Allocate()
```

Comment:
[This One](https://github.com/dotnet/roslyn/pull/17134/files/086a13d34e0307158627d4c819fc565441e0108a#diff-ebcd664962405d1d4587e31b0b615dd2R16)